### PR TITLE
Remove SAP Cloud Business Application Tools for Eclipse

### PR DESCRIPTION
**Pull request by mofterdinger**
The SAP Cloud Business Application Tools for Eclipse are no longer maintained and the last available version will be removed from the external Eclipse Update site: https://tools.hana.ondemand.com/

For René:

- [ ] Remove also from Help Portal to avoid dead links

### CODE REVIEW

**Comment by mofterdinger** [comment]
@renejeglinsky Hi Rene, I don't understand the build error. I couldn't find any reference to `#eclipse` in my PR branch. Do you have an idea ?
Thanks,
Markus

File: get-started/troubleshooting.md [hunk 1]
```diff
@@ -236,7 +236,7 @@ In addition you might want to remove the H2 dependency, which is included in the
 
 If you don't want to exclude dependencies completely, but make sure that an in-memory H2 database **isn't** used, you can disable Spring Boot's `DataSource` auto-configuration, by annotating the `Application.java` class with `@SpringBootApplication(exclude = org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration.class)`. In that mode CAP Java however can still react on explicit data source configurations or database bindings.
 
-### What to Do About Maven-Related Errors in Eclipse's Problems View? { #eclipse}
```
Comment (smahati) [inline get-started/troubleshooting.md:239]: @mofterdinger maybe it's related to this?

Comment (smahati) [reply get-started/troubleshooting.md:239]: Hmm there are list of files where the references are supposed to be from, I'll check then individually and try to find them :)

Comment (smahati) [reply get-started/troubleshooting.md:239]: found one stray, I guess we need to fix that one first: https://github.tools.sap/cap/docs/blob/d9e5be4eb1571fee0d35edd956c2b1c5e4a80e76/java/multitenancy.md?plain=1#L840

Comment (mofterdinger) [reply get-started/troubleshooting.md:239]: will be remove by this PR: https://github.tools.sap/cap/docs/pull/510

**Comment by mofterdinger** [comment]
Hi, can this PR be merged ?

File: get-started/troubleshooting.md (hunk 1 above)
Comment (renejeglinsky) [reply get-started/troubleshooting.md:239]: The solution here doesn't use any functionality the eclipse plugin provided?

Comment (mofterdinger) [reply get-started/troubleshooting.md:239]: no, it just uses features of the maven plugin in eclipse which is still installed.

**Review by renejeglinsky — APPROVED** [review]

**Pull request merged**